riskified-sdk/src/main/resources/riskified_sdk.properties

```diff
@@ -1,21 +1,22 @@
 # Your shop domain as registered to Riskified
-shopUrl=
+shopUrl=roomstogo.com
 
 # Your authentication token string
-authKey=
+authKey=c47b2a9090b9db57596fa2394f40e202
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |

 <!-- c45286dc-93b7-4a9a-9d90-31b7026f4f8e -->